knative-operator/hack/update-manifests.sh

```diff
@@ -78,5 +78,8 @@ git apply "$root/knative-operator/hack/012-eventing-kafkachannel-addressable-res
 # SRVKE-1184 Migration to Broker components
 git apply "$root/knative-operator/hack/014-eventing-kafka-lease-name-remapping.patch"
 
+# SRVKE-1202 temporary fix for a missing ConfigMap knative-eventing/config-kafka-source-defaults
+git apply "$root/knative-operator/hack/017-eventing-kafka-configmap-knative-eventing-config-kafka-source-defaults.patch"
+
```

Comment:
I'm calling it 017 and adding this patching here to not conflict with the other PR https://github.com/openshift-knative/serverless-operator/pull/1498

Reply:
yeah, all cool w/ the numbering :-)